docs/debugging-openshift.md

```diff
@@ -25,6 +25,41 @@ System Environment
 
   Containers need to be able to resolve hostnames, so if you run a local DNS server on your host, you should update your /etc/resolv.conf to instead use a DNS server that will be reachable from within running containers.  Google's "8.8.8.8" server is a popular choice.
 
+Diagnostics
+-----------
+
+In any system like OpenShift where a lot of components are working together, 
+it can be difficult to figure out where a process is breaking down. To help
+with this discovery and diagnosis, `openshift ex diagnostics` can detect a
+number of common problems and misconfigurations. It can diagnose a running
+master / node, client connection, or any combination of these that is detected
+or specified. For any (potential) problems that it detects, it will explain
+in detail what is going on.
+
+    $ openshift ex diagnostics
```

Comment:
> It's both! And in this case I kind of want to emphasize that it's experimental AKA alpha

If you aren't ready for it to be exposed, don't expose it as `openshift-diagnostics`.  I don't have a preference one way or the other, but things expose as main commands are referenced by their official names and only live under `ex` for compatibility until we clean them up.